ares/ws/apu/serialization.cpp

```diff
@@ -19,6 +19,8 @@ auto APU::serialize(serializer& s) -> void {
   s(io.seqDbgOutputForce55);
   s(io.seqDbgChForce4);
   s(io.seqDbgChForce2);
+  s(io.seqDbgNoise);
```

Comment:
I assume there's no need to bump the serialization version within the scope of commits in one release?

Reply:
Should be fixed now!